Update to the name function urllib.urlopen

tacigomess opened the pull request:
According to the https://docs.python.org/3/library/urllib.request.html#module-urllib.request the name of the function has been changed
to urllib.request.urlopen

tacigomess committed:
Updating weather-api for python3

The weather-api name has change in python3 to python3-weather-api.

tacigomess committed:
Update to the name function urllib.urlopen

According to the https://docs.python.org/3/library/urllib.request.html#module-urllib.request the name of the function has been changed
to urllib.request.urlopen

tacigomess committed:
Update to the name function urllib.urlopen

According to the https://docs.python.org/3/library/urllib.request.html#module-urllib.request the name of the function has been changed
to urllib.request.urlopen

satyammittal commented on google_places.py:
```diff
@@ -41,7 +41,7 @@ def get_location():
     #  u'country_code': u'IN', u'country_name': u'India', u'postal': u'500025'}
     url = "https://geoip-db.com/json"
-    response = urllib.urlopen(url)
+    response = urllib.request.urlopen(url)
```
You have to directly import module `urllib.request` to make it work.

satyammittal committed:
Merge pull request #20 from tacigomess/fix-weather-api

Updated requirement for python3 support.

arxcruz commented:
Instead of use urllib, just use requests as in the other pieces here.

tacigomess committed:
Remove Urllib

Instead to use Urllib use direct requests

tacigomess committed:
Remove Urllib

Instead to use Urllib use direct requests

tacigomess committed:
raw_input was replaced to input()

Python3 use input() instead raw_input()

arxcruz committed:
Enable tox support

This patch enable tox support on chatbot project.
Initially, it's skipping several pep8 otherwise it won't pass the CI.
Once the patch get merged, it's possible to start remove the ignored
peps.
This will allow tests to be executed locally, before the patch get
submited, and improve the code quality.
This also add the test-requirements.txt and change the travis yaml file
to use it, instead of the pip install commands.

satyammittal committed:
Merge pull request #23 from tacigomess/fix-raw_input

raw_input was replaced to input()

satyammittal committed:
Merge pull request #24 from arxcruz/enable-tox

Enable tox support

tacigomess committed:
Merge branch 'fix-urllib.urlopen' of https://github.com/tacigomess/voice-enabled-chatbot into fix-urllib.urlopen

satyammittal reviewed (changes requested):
Please update branch with latest changes.
